gobcore/typesystem/gob_geotypes.py

```diff
@@ -88,7 +88,7 @@ def from_value(cls, value, **kwargs):
         if isinstance(value, str):
             regex = re.compile("^POINT\s*\([0-9\s\.]*\)$")
             if not regex.match(value):
-                raise ValueError("Illegal WKT value")
+                raise ValueError("Illegal Point WKT value")
```

Comment:
Maybe also add the value in the error message, just as with Polygon and Geometry